packages/react-router-redux/examples/ssr/README.md

```diff
@@ -0,0 +1,103 @@
+# Server-side Rendering
+
+Server-side rendering doesn't have to be difficult and RRR5 makes this _really_ simple. If you're wanting a detailed tutorial on how to implement server-side rendering using `create-react-app`, be sure to visit this [this post](https://medium.com/@cereallarceny/server-side-rendering-with-create-react-app-fiber-react-router-v4-helmet-redux-and-thunk-275cb25ca972) and [this repository](https://github.com/cereallarceny/cra-ssr) for an example.
+
+## Client
+
+Your application entry point (in this case, `client.js`) is fairly standard. You hook up to a store (`client-store.js`) that loads your reducers (`reducers.js`) and load in your main entry component (`<App>`).
+
+You can opt to load whatever components you want inside of `ConnectedRouter`. In the example above, we load a single `<App>` component which handles all routing to other files there. Alternatively, you may declare all your routes in this file, leaving it up to the developer's organizational discretion. If you have a single component (in this case, `<App>`) then it will make any server-side rendering logic much simpler. You will not need to update your routes on both the client and the server, but simply just the client. See the "Server" below for more details.
+
+## Server
+
+Usage on the server (for server-side rendering) is relatively simple as well. There's no need to distinguish between `BrowserRouter` and `StaticRouter` since `ConnectedRouter` will handle this distinction. Setting up a full Express/Node application to run your server is beyond the scope of this documentation. For a tutorial using `ConnectedRouter` on both the client and server side, please see [this post](https://medium.com/@cereallarceny/server-side-rendering-with-create-react-app-fiber-react-router-v4-helmet-redux-and-thunk-275cb25ca972) and [this repository](https://github.com/cereallarceny/cra-ssr) for an example.
```

Comment:
> There's no need to distinguish between `BrowserRouter` and `StaticRouter` since `ConnectedRouter` will handle this distinction.

I might reword this a bit to mention that the different is the type of history passed to the `ConnectedRouter`.

> Just like on the client, we will use a `ConnectedRouter`. The only difference is that we will pass it a memory history instance instead of a browser history.